### PR TITLE
Migrate search menu, dropdown, and svgicons

### PR DESCRIPTION
**Pull request by cehsu**
These changes migrate the Search Menu (and its children components) into the design system. The toggle on blur behavior has been made more consistent with how this behavior is handled elsewhere, such as in the errorable checkbox.

### CODE REVIEW

File: dist/styles.bundle.js [hunk 1]
```diff
@@ -63,15 +63,15 @@
...
... failed: ModuleBuildError: Module build failed: \n  @include focus;\n          ^\n      No mixin named focus\n\nBacktrace:\n\tnode_modules/uswds/src/stylesheets/core/_base.scss:34\n      in /Users/clairehsu/Projects/design-system/node_modules/uswds/src/stylesheets/core/_base.scss (line 34, column 12)\n    at runLoaders (/Users/clairehsu/Projects/design-system/node_modules/webpack/lib/NormalModule.js:192:19)\n    at /Users/clairehsu/Projects/design-system/node_modules/loader-runner/lib/LoaderRunner.js:364:11\n    at /Users/clairehsu/Projects/design-system/node_modules/loader-runner/lib/LoaderRunner.js:230:18\n    at context.callback (/Users/clairehsu/Projects/design-system/node_modules/loader-runner/lib/LoaderRunner.js:111:13)\n    at Object.asyncSassJobQueue.push [as callback] (/Users/clairehsu/Projects/design-system/node_modules/sass-loader/lib/loader.js:55:13)\n    at Object.<anonymous> (/Users/clairehsu/Projects/design-system/node_modules/async/dist/async.js:2257:31)\n    at Object.callback (/Users/clairehsu/Projects/design-system/node_modules/async/dist/async.js:958:16)\n    at options.error (/Users/clairehsu/Projects/design-system/node_modules/node-sass/lib/index.js:294:32)");
```
Comment (cehsu) [inline dist/styles.bundle.js]: I was able to generate this file but had some trouble generating the css files in the design system’s dist folder after making changes. Are there any steps besides running `npm run start` to go about this?

Comment (jbalboni) [reply dist/styles.bundle.js]: I think this has been broken for a little bit. I pushed a fix to master, it should work if you pull from master and try again.

Comment (cehsu) [reply dist/styles.bundle.js]: Thanks @jbalboni, the styles file is now being generated.

File: build/components/raw/search_menu/SearchMenu.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import SearchMenu from './SearchMenu.jsx';
+
+describe('<SearchMenu/>', () => {
+  it('should render', () => {
+    const tree = shallow(<SearchMenu id="modal" title="Modal title" visible onClose={() => {}}>Modal contents</SearchMenu>);
+    expect(tree.text()).to.contain('Modal contents');
+  });
+
+  it('should pass aXe check', () => {
```
[the remark this answers is not in the document]
Comment (cehsu) [reply build/components/raw/search_menu/SearchMenu.unit.spec.jsx]: Great idea @jbalboni, test added.

File: build/components/raw/search_menu/SearchMenu.unit.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import { axeCheck } from '../../../lib/testing/helpers';
+
+import SearchMenu from './SearchMenu.jsx';
+
+describe('<SearchMenu/>', () => {
+  it('should render', () => {
+    const tree = shallow(<SearchMenu id="modal" title="Modal title" visible onClose={() => {}}>Modal contents</SearchMenu>);
+    expect(tree.text()).to.contain('Modal contents');
```
[the remark this answers is not in the document]
Comment (cehsu) [reply build/components/raw/search_menu/SearchMenu.unit.spec.jsx]: Good catch, fixed.

**Comment by cehsu** [comment]
Thanks @jbalboni, I have created an [epic](https://github.com/department-of-veterans-affairs/design-system/issues/20) for documenting child components, and issues for those used here as well (linked in the epic).

**Pull request merged**